pkgs/development/rocm-modules/6/rocm-device-libs/default.nix

```diff
@@ -19,39 +22,44 @@ let
 in
 stdenv.mkDerivation (finalAttrs: {
   pname = "rocm-device-libs";
-  version = "6.0.2";
+  # In-tree with ROCm LLVM
+  inherit (rocm-merged-llvm) version;
+  src = rocm-merged-llvm.llvm-src;
 
-  src = fetchFromGitHub {
-    owner = "ROCm";
-    repo = "ROCm-Device-Libs";
-    rev = "rocm-${finalAttrs.version}";
-    hash = "sha256-7XG7oSkJ3EPWTYGea0I50eB1/DPMD5agmjctxZYTbLQ=";
-  };
+  postPatch = ''
+    cd amd/device-libs
+  '';
 
   patches = [ ./cmake.patch ];
 
   nativeBuildInputs = [
     cmake
-    rocm-cmake
+    ninja
+    python3
   ];
 
-  buildInputs = [ libxml2 ];
-  cmakeFlags = [ "-DLLVM_TARGETS_TO_BUILD=AMDGPU;${llvmNativeTarget}" ];
+  buildInputs = [
+    libxml2
+    zlib
+    zstd
+    ncurses
+    rocm-merged-llvm
+  ];
 
-  passthru.updateScript = rocmUpdateScript {
-    name = finalAttrs.pname;
-    owner = finalAttrs.src.owner;
-    repo = finalAttrs.src.repo;
-  };
+  dontStrip = true;
+  env.CFLAGS = "-g1 -gz";
```

Comment:
Is there a reason these flags are added?

Reply:
Will yeet them soontm, there's an entry in the todos.  
I added line table only debug flags to a lot of the derivations while profiling some perf issues while this was out of tree.

Reply:
yote